modules/dnn/src/darknet/darknet_importer.cpp

```diff
@@ -0,0 +1,202 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
I think we need to replace it. Please take a look on more fresh versions of the licence.

Reply:
@AlexeyAB, I don't think that we should place past years copyrights to newly added source.
@vpisarev, please tell us if the following header is enough?
```

// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
//
// Copyright (C) 2017, Intel Corporation, all rights reserved.
// Third party copyrights are property of their respective owners.
```
Or the same as above but with a current date.